README.md

```diff
@@ -83,6 +83,14 @@
 <img src="./screenshots/QU11.png"/>
 <img src="./screenshots/QU12.png"/>
 <img src="./screenshots/QU13.png"/>
+##Screenshots of Jointures table 
+<img src="./screenshots/Jointure_1.png"/>
+<img src="./screenshots/Jointure_2.png"/>
+<img src="./screenshots/Jointure_3.png"/>
+<img src="./screenshots/Jointure_4.png"/>
+<img src="./screenshots/Jointure_5.png"/>
+<img src="./screenshots/Jointure_6.png"/>
+<img src="./screenshots/Jointure_7.png"/>
```

Comment:
I can see you added the images of your queries to your readme file, but let's remember that the requirement is that the images are added to your PR description😉.
<img width="607" alt="image" src="https://github.com/ridabensalem/vet-clinic/assets/64643995/c46d920f-3d92-43ed-9af5-969fec03eb7c">
kindly edit your PR description and add all the necessary images showing each queries and the result in the terminal😉.